toolkit/tools/srpmpacker/srpmpacker.go

```diff
@@ -456,7 +457,7 @@ func specsToPackWorker(requests <-chan string, results chan<- *specState, cancel
 		containingDir := filepath.Dir(specFile)
 
 		// Find the SRPM that this SPEC will produce.
-		defines := rpm.DefaultDefines()
+		defines := rpm.DefaultDefines(*runCheck)
```

Comment:
Same comment as in `specreader.go` for this line and the later in the code.

Reply:
Thanks- redid the PR to pass through the runCheck variable as function args all around.